cli/cmd/volume.go

```diff
@@ -75,8 +83,29 @@ var volumeCreateCmd = &cobra.Command{
 	Short: helpVolumeCreateCmd,
 	Run: func(cmd *cobra.Command, args []string) {
 		validateNArgs(cmd, 2, 0)
-		volname := cmd.Flags().Args()[0]
-		bricks := cmd.Flags().Args()[1:]
+		force := cmd.Flags().Args()[0]
+		volname := cmd.Flags().Args()[1]
+		bricks := cmd.Flags().Args()[2:]
+		force_, err := strconv.ParseBool(force)
```

Comment:
This is not required, already available as flag. as mentioned in previous comment

Reply:
Removed.